lib/src/components/Drawer/AssetDrawer/index.tsx

```diff
@@ -43,8 +51,9 @@ export const AssetDrawerComponent = forwardRefWithAs<AssetDrawerProps, 'div'>(
         hideOnInteractOutside={hideOnInteractOutsideFn}
         placement="bottom"
         ref={ref}
-        size={null}
         store={store}
+        // Needed to override the default height set in drawer.module.scss
+        style={{ '--drawer-height': 'calc(100% - 3rem)' } as React.CSSProperties}
```

Comment:
You could override it directly in the `asset-drawer` class
.asset-drawer {
...css
--drawer-height: calc(100% - 3rem)
}